s3fs-2021.11.1.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
s3fs-2021.11.1.dist-info/LICENSE.txt,sha256=3DWZ-ma8G8_6I2g4vi-04V-EzkgBJE7sk9kOUVz8WNE,1505
s3fs-2021.11.1.dist-info/METADATA,sha256=ug5Xoa-_Xr-PhxtQbtf99HgWMCAAmoFLlkreXi_tYtw,1564
s3fs-2021.11.1.dist-info/RECORD,,
s3fs-2021.11.1.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
s3fs-2021.11.1.dist-info/WHEEL,sha256=OqRkF0eY5GHssMorFjlbTIq072vpHpF60fIQA6lS9xA,92
s3fs-2021.11.1.dist-info/pbr.json,sha256=0sInMIvBRipDYl53OSOltXEZ_22FlQwyqy3jrGJehmk,46
s3fs-2021.11.1.dist-info/top_level.txt,sha256=Lf6EI3TdjlPu7TN-92IIY6c-GdrHnkrKGb1n0iwKooI,5
s3fs/__init__.py,sha256=_6_Vs_vblhJSJw-62JVIBIM8kTKLhuwPFAIvt3hanls,160
s3fs/__pycache__/__init__.cpython-39.pyc,,
s3fs/__pycache__/_version.cpython-39.pyc,,
s3fs/__pycache__/core.cpython-39.pyc,,
s3fs/__pycache__/errors.cpython-39.pyc,,
s3fs/__pycache__/mapping.cpython-39.pyc,,
s3fs/__pycache__/utils.cpython-39.pyc,,
s3fs/_version.py,sha256=xitrjfl3pQdUyYTbpZBurMqhs1ZfNSoNI8Yq_FSG5fI,501
s3fs/core.py,sha256=3o_0WTCln5as-ikFFMOi0BODBuLHC5gBPdg4da7gz2A,72588
s3fs/errors.py,sha256=y4lpQHKdnuO4_tmiob_BQc_FcyrDKrA7GQZLRz7VDiY,7712
s3fs/mapping.py,sha256=FoqEdMne7LXUL4HgPN4j6WsMsrwxpb53GynDhXs9VRI,237
s3fs/utils.py,sha256=lwit-la3cn_-4Ic0nOFug7G9M8aBO2T0Rdpb57wp2Oc,5765
